src/Components/ServiceScene/Breakdowns/ByFrameRepeater.tsx

```diff
@@ -49,8 +49,12 @@ export class ByFrameRepeater extends SceneObjectBase<ByFrameRepeaterState> {
       const data = sceneGraph.getData(this);
 
       this._subs.add(
-        data.subscribeToState((data) => {
-          if (data.data?.state === LoadingState.Done) {
+        data.subscribeToState((data, prevData) => {
+          if (
+            data.data?.state === LoadingState.Done ||
```

Comment:
Maybe just != Error and we account for any change we make? E.g. we go back to Loading for sharding, etc.

Reply:
I mean, it's either Done, or Loading, or Streaming, or else Error. Not a big deal in any case.

Reply:
Plus with sharding we can have data we want to viz and errors?